storage/simulation/cluster.go

```diff
@@ -64,6 +69,8 @@ func createCluster(stopper *stop.Stopper, nodeCount int) *Cluster {
 		nodes:         make(map[proto.NodeID]*Node),
 		stores:        make(map[proto.StoreID]*Store),
 		ranges:        make(map[proto.RangeID]*Range),
+		rand:          rand,
+		seed:          seed,
```

Comment:
noted.  My first pass is actually going to only be single threaded.  I have plans on having a separate goroutine for each store (hence all the locking), but that will be for later.